language/diem-framework/modules/0L/Globals.move

```diff
@@ -111,8 +117,8 @@ module Globals {
           // uses "scaled representation", since there are no decimals.
           subsidy_ceiling_gas: 8640000 * coin_scale, // subsidy amount assumes 24 hour epoch lengths. Also needs to be adjusted for coin_scale the onchain representation of human readable value.
           vdf_difficulty: 5000000, // FYI approx 10 mins per proof on 2020 macbook pro 2.5 ghz quadcore
-          epoch_mining_thres_lower: 20,
-          epoch_mining_thres_upper: 240, // upper bound enforced at 6 mins per proof.
+          epoch_mining_thres_lower: 7, // NOTE: bootstrapping, allowance for operator error.
+          epoch_mining_thres_upper: 72, // upper bound enforced at 20 mins per proof.
```

Comment:
I thought we wanted to change this to 52. Upper bound enforced at 27 mins right

Reply:
I think we need a wider range at the start to see how much variation we get.